Formula/socket_vmnet.rb

```diff
@@ -16,7 +16,7 @@ class SocketVmnet < Formula
     sha256 cellar: :any_skip_relocation, big_sur:        "ed57ff746e0dca70e6d38564dc4158ef6034726c601493cffcc0c765576b6b7f"
   end
 
-  keg_only "#{HOMEBREW_PREFIX}/bin is often writable by a non-admin user"
+  keg_only "the bin directory is often writable by a non-admin user"
```

Comment:
```suggestion
  keg_only "Homebrew's bin directory is often writable by a non-admin user"
```
Though, repeating my comment from Homebrew/brew#14996,
> Yea, we could do that. Not even sure the `keg_only` reason is worth it, though -- if an attacker has write access to `$HOMEBREW_PREFIX/bin`, you're already hosed. But CC `@Homebrew/maintainers` for thoughts.

I'm also not sure removing this means we don't need an audit. But removing it does allow us to punt on it.

Reply:
If we decide the keg-only reason isn't good enough:
```suggestion
```

Reply:
It still needs the `keg_only` doesn't it, just not the reason?

Reply:
Well, if the reason isn't compelling, then there's no reason it should be keg-only, right?